dc_utils/templates/dc_base.html

```diff
@@ -7,15 +7,15 @@
             <p><a class="ds-skip-link" href="#main">skip to content</a></p>
 
             {% block header_base %}
-                <header role="banner">
-                    <div class="container">
-                        {% if site_logo %}
-                            <a href="/">
-                                <img src="{% static site_logo %}" alt="{{ site_title }}" width="{{ SITE_LOGO_WIDTH|default:"100" }}">
-                            </a>
-                        {% endif %}
-                        {% block site_menu %}{% endblock site_menu %}
-                    </div>
+                <header class="ds-header">
+                    {% if site_logo %}
+                        <a href="/">
+                            <img src="{% static site_logo %}" alt="{{ site_title }}" width="{{ SITE_LOGO_WIDTH|default:"100" }}">
+                            <span>{{ site_title }}</span>
```

Comment:
Will I need to adapt context processors in other products to make use of `site_title` as shown here?

Reply:
Given the blocking is basically the same and the reference to site_title is also the same I can't see that there would be any changes required in other projects - but I'm happy to be corrected!